src/routes/MainRoute/MainRoute.jsx

```diff
@@ -7,20 +32,157 @@ const MainRoute = () => {
-  };
-
   return (
-    <div>
-      <h2>MainPage</h2>
-      <button type="button" onClick={handleRegisterOpen}>
-        Sign up
-      </button>
-      <button type="button" onClick={handleLoginOpen}>
-        Login
-      </button>
-    </div>
+    <MainPageLayout>
+      <Header />
+      <MainPageSectionOne>
+        <Typography
+          variant="h1"
+          sx={{
+            fontSize: '64px',
+            lineHeight: '76.8px',
+            color: 'var(--color-dark)',
+          }}
+        >
+          <b>Talk</b> Without Borders
+        </Typography>
+        <Typography
+          variant="h1"
+          sx={{
+            fontSize: '64px',
+            lineHeight: '76.8px',
+            color: 'var(--color-dark)',
+          }}
+        >
+          <b>Travel</b> Without Limits
+        </Typography>
+        <Typography
+          sx={{
+            marginTop: '24px',
+            fontSize: '20px',
+            lineHeight: '24px',
+          }}
+        >
+          Start chatting now and be a <br /> part of the global conversation!
+        </Typography>
+        <BasicButton
+          variant="contained"
+          color="primary"
+          sx={{
+            margin: '32px',
+          }}
+          text="Start chatting"
+          handleClick={handleRegisterOpen}
+        />
+        <SVGImage1 src={SVG1} alt="SVG1" />
+        <SVGImage2 src={SVG2} alt="SVG2" />
+        <SVGImage3 src={SVG3} alt="SVG3" />
+        <SVGImage4 src={SVG4} alt="SVG4" />
+      </MainPageSectionOne>
+      <MainPageSectionTwo />
+      <MainPageSectionThree id="about">
+        <SectionThreeContainer display="flex">
+          <Typography
+            variant="h4"
+            sx={{
+              marginBottom: '20px',
+              fontSize: '40px',
+              lineHeight: '48px',
+            }}
+          >
+            What is <b>Talk & Travel?</b>
+          </Typography>
+          <Typography
+            sx={{
+              fontSize: '20px',
+              lineHeight: '24px',
+            }}
+          >
+            Talk & Travel is the chat where everyone joins a vibrant community,
+            connecting with fellow travelers to share tips, stories, and
+            experiences from around the world.
+          </Typography>
+          <Typography
+            sx={{
+              fontSize: '20px',
+              lineHeight: '24px',
+            }}
+          >
+            <br />
+            Join specific countries or chat directly with your fellow explorers.
+            Sign up now and start enjoying the journey!
+          </Typography>
+          <BasicButton
+            sx={{ marginTop: '40px' }}
+            variant="outlined"
+            text="Get started"
+            handleClick={handleRegisterOpen}
+          />
+        </SectionThreeContainer>
+      </MainPageSectionThree>
+      <MainPageSectionFour id="benefits">
+        <SectionFourContainer>
+          <Bubble
+            marginbottom="20px"
```

Comment:
In otder to avoid such long repeatable content which is static also, you could create a varialble to store all the info you need as an array and render it with array method 'map' or something.

Reply:
fixed.